src/ekobox/operators_card.py

```diff
@@ -7,27 +7,27 @@
 from ekomark.data import operators
 
 
-def gen_op_card(Q2grid, update=None, name=None):
-    """Generate operators card.
+def generate(Q2grid, update=None, name=None):
+    """
+    Generate operators card.
 
-    Generates an operator card with some mandatory user choice
+    Generates an operators card with some mandatory user choice
     (in this case only the Q2 grid) and some default values which
     can be changed by the update input dict.
 
     Parameters
     ----------
-    Q2grid : list(float)
-        grid for Q2
-    update : dict
-        dictionary of info to update in op. card
-    name : str
-        name of exported op.card (if name not None)
+        Q2grid : list(float)
```

Comment:
Again: this indentation and the above newline for the first sentence is non-standard.

Please, follow the referene: https://www.sphinx-doc.org/en/master/usage/extensions/example_numpy.html

Reply:
as you maybe noticed, in pineko I don't indent, but here I wanted to stick to the scheme - you know this is due to VS code, however I'm not strongly opinioned any longer (since never using); still do you want to mix conventions?

Reply:
If there is already a convention, and you start applying a new one, or you spend a lot of energy in enforcing it (manually checking or spending the time to automate), or it will be broken at some point.

Since there is a convention, and it has not been decided by us, let's stick to it, with an incremental transition.